File: 8-Kyu/java-codewars/reverse-string.java
/**
 
Complete the solution so that it reverses the string passed into it.

'world'  =>  'dlrow'
'word'   =>  'drow'

 */

 public class Kata {

  public static String solution(String str) {
    // Your code here...
    String result = new StringBuffer(str).reverse().toString();
     return result;
  }

}

/**
 
Input: String
Output: String 

Steps:
- create a new variable the stores the solution
- use StringBuffer to manipulate the string 
- .reverse() to reverse the string
- .toString to convert it back to a string value again
- return new variable where you manipulated the value from

 */

File: 8-Kyu/python-codewars/reverse.string.py
"""
PROMPT:

Complete the solution so that it reverses the string passed into it.

'world'  =>  'dlrow'
'word'   =>  'drow

"""

def solution(string):
    reverse_str = ''.join(reversed(string))
    return reverse_str

"""

Input: String
Output: String

Steps
- Create a new variable that stores result 
- Use .join() and '' to separate each character
- .reversed() function to reverse in order the string
- Return the variable the stores the result

"""